chain/stmgr/supply.go

```diff
@@ -443,7 +443,8 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 			builtin.IsPaymentChannelActor(actor.Code) ||
 			builtin.IsEthAccountActor(actor.Code) ||
 			builtin.IsEvmActor(actor.Code) ||
-			builtin.IsPlaceholderActor(actor.Code):
+			builtin.IsPlaceholderActor(actor.Code) ||
+			builtin.IsDatacapActor(actor.Code):
```

Comment:
We should probably exclude this actor, right? The funds cannot be withdrawn.

Reply:
Add this to the `unCirc` check above:

`a == builtin.DatacapActorAddr`

Reply:
I wasn’t sure… cuz it is not supposed to receive funds but it can . I don’t have a strong opinion - your suggestion would make senses cuz I don’t think the actor has a withdraw fil function. Will confirm and make the change

Reply:
(I hope...)

Reply:
ah okay good to know! thanks  & updated